doc/content.tex

```diff
@@ -1,677 +0,0 @@
-    $q$ is listed in \ref{eq:ma-model}.
-
-    \begin{gather} \label{eq:ma-model}
-        y_t = \theta_1 \epsilon_{t-1} + \theta_2 \epsilon_{t-2} + \dots + \theta_p y_{t-q} + \epsilon_t \\ \nonumber
-        \epsilon_t \overset{iid}{\sim} N(0, \sigma^2)
-    \end{gather}
-
-    The last part of the model is used when a time series is non stationary. There are several ways how to make a
-    particular time series stationary. Box\,--\,Jenkins methodology uses differencing\,--\,integration part I(d).  The
-    order of differentiated original series is denoted by $d$ letter. Usually first order differences are enough to
-    make a time series stationary.
-
-    It is important to mention that models AR(p) and MA(q) are invertible. Therefore any stationary AR(p) model can
-    be written as MA($\infty$) and with some assumptions vice versa \cite{brockwell}. ARIMA model is often written
-    with backshift operator $By_t=y_{t-1}$. With this operator \\ARIMA(p, d, q) is listed in \ref{eq:arima}. On the
-    left side of the equation is AR(P) process and on the right MA(q).
-
-    \begin{gather} \label{eq:arima}
-        (1- \phi_1B - \dots - \phi_pB^p)(1-B)^d y_t = \\ \nonumber
-         c + (1+\theta_1B+\dots+\theta_qB^q) \epsilon_t
-    \end{gather}
-
-    The parameters of the model, including AR and MA part can by estimated by non-linear optimization algorithm
-    with likelihood as objective function \cite{brockwell}. However this function is much more complicated than for
-    exponential smoothing models. For successful estimation a certain number of historical points needs to
-    be available. Reference \cite{cipra} advises minimal training size of at least fifty observations.
-
-    After this theoretical part it is clear that ARIMA models are more complicated than family of moving averages.
-    To this also refers \cite{hyndman-forecasting} and adds that ARIMA models are less robust than exponential
-    smoothing models.
-
-    %%%%%%%%
-    \section{Artificial Neural Networks} \label{sec:ann}
-    Recently a large number of successful applications using neural networks for time series modeling show that they
-    can produce valuable results \cite{ann-forecasting-state-art}. There are several non trivial issues with
-    determining the appropriate architecture of the network. This has to be taken into account because it can
-    dramatically effect learning performance and forecasting accuracy \cite{ann-model-selecting}.
-    Besides the problems with selecting right architecture learning process of artificial neural network is much more
-    computationally expensive than selecting appropriate ARIMA or exponential smoothing model \cite{ann-forecasting}.
-
-    Because Hawkular forecasting engine should be capable of predicting thousands of metrics at the same time, models
-    based on neural networks would have too high computational requirements. Therefore they are not suitable for our
-    environment.
```

Comment:
Tazke: ``requirements. Therefore they are not suitable for our  environment.``
Alebo si chcela povedat: `requirements, therefore, they are not suitable for our  environment.``` ?